server/src/main/java/org/uiautomation/ios/utils/SimulatorSettings.java

```diff
@@ -285,47 +286,92 @@ public void resetContentAndSettings() {
         System.err.println("couldn't re-create: " + contentAndSettingsFolder);
       }
     } else {
-      // Starting with Xcode 6 and later, we can use simctl to do the hard work for us.
-      // If it fails, we have to turn the device off first.
       if (!eraseSimulator()) {
-        log.log(Level.WARNING, "Reset content and settings failed, " +
-            "possibly device is in booted state, shutdown device = " + deviceUUID);
-        List<String> simctlArgs = new ArrayList<>();
-        simctlArgs = Arrays.asList("xcrun", "simctl", "shutdown", deviceUUID);
-        Command simctlCmd = new Command(simctlArgs, true);
+        log.info("Erase contents and settings failed on this device: " + deviceUUID);
```

Comment:
What I am thinking is executing the command `xcrun simctl erase <deviceID>` in this case will mostly fail if the simulator needs time to completely shut down. It will need some attention if we wait for a while, try to shut down the simulator again, but the command is still failed.